src/context/GameContext.tsx

```diff
@@ -260,24 +261,30 @@ export const GameContextProvider = (props: { children: React.ReactNode }) => {
       transition: "TRANSITION_FROM_GAME_OVER",
     });
   };
+  const { width } = useWindowSize();
+  const isMobile = width < 1024;
 
-  const shareSupported = Boolean(
-    "canShare" in navigator && navigator.canShare(SHARE_DATA)
-  );
+  const shareSupported =
+    isMobile &&
+    Boolean("canShare" in navigator && navigator.canShare(SHARE_DATA));
 
   const { clipboardSupported, copy, copied } = useClipboard();
 
   const handleCopyLink = () => {
-    if (shareSupported) {
-      navigator.share(SHARE_DATA).catch((error) => {
-        console.error("Error sharing:", error);
-      });
+    if (isMobile) {
+      if (shareSupported) {
```

Comment:
Why is this `if` statement nested if both `else` statement results in the same logic?